client/state/data-layer/wpcom/activity-log/rewind/index.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { pick } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	REWIND_STATUS_REQUEST,
+} from 'state/action-types';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { http } from 'state/data-layer/wpcom-http/actions';
+
+const fetchRewindStatus = ( { dispatch }, action ) => {
+	dispatch( http( {
+		method: 'GET',
+		path: `/activity-log/${ action.siteId }/rewind`,
+	}, action ) );
+};
+
+const fromApi = response => ( {
+	active: response.use_rewind,
+	firstBackup: response.first_backup_when,
+	error: { message: response.error },
+} );
+
+const updateRewindStatus = ( { dispatch }, { siteId }, next, data ) => {
+	dispatch( {
+		type: REWIND_STATUS_REQUEST,
```

Comment:
...passing the `meta` along with a subsequent dispatch does not have good results.

So, I'm fine with using different action types, but the docs were leading me to think they could be re-used, for example:

> Because the information from the request response extends the action through meta, we can actually re-use the originating action and handle it based on that meta.